src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs

```diff
@@ -102,6 +104,148 @@ public TransceiverDirection TransceiverDirection
             }
         }
 
+        /// <summary>
+        /// Gets or sets the transceiver codec of current media source.
+        /// </summary>
+        /// <remarks>
+        /// This API is not supported in <see cref="MediaFileSource"/>, <see cref="MediaPacketSource"/>.<br/>
+        /// The WebRTC must be in the <see cref="WebRTCState.Idle"/> state when transceiver codec is set.
+        /// </remarks>
+        /// <value>A <see cref="TransceiverCodec"/> that specifies the transceiver codec.</value>
```

Comment:
there is no benefit from linking to a property type, the link to the type is inserted automatically, but it can stay

Reply:
Removed.